doc/api/crypto.md

```diff
@@ -2141,6 +2139,95 @@ threadpool request. To minimize threadpool task length variation, partition
 large `randomFill` requests when doing so as part of fulfilling a client
 request.
 
+### crypto.scrypt(password, salt, keylen[, options], callback)
+<!-- YAML
+added: REPLACEME
+-->
+- `password` {string|Buffer|TypedArray}
+- `salt` {string|Buffer|TypedArray}
+- `keylen` {number}
+- `options` {Object}
+  - `N` {number} CPU/memory cost parameter. Must be a power of two greater
+                 than one. **Default:** `16384`
+  - `r` {number} Block size parameter. **Default:** `8`
+  - `p` {number} Parallelization parameter. **Default:** `1`
```

Comment:
For better or worse, they're the "official" names of the algorithm's parameters.  I'd say straying from the established nomenclature is not a good thing.

Reply:
@bnoordhuis I don’t think the typical consumer of these APIs would be familiar with the ‘official’ nomenclature, though – it would be good to mention the names `N`, `r`, `p` in the docs, but I do think that our API should have names that make sense to most developers rather than crypto people?

Reply:
I did consider more expressive names but if you survey scrypt bindings in other languages (java, python, php, etc.), they all use N, r and p.  We'd be the outlier.

To be fair, the perl binding doesn't call them that but it uses CPU time as the bound, not algorithmic complexity.

Reply:
To be honest, I am doing the same thing in my PQC implementations in C, but I feel like node's crypto API could be more accessible if we use more intuitive names. On the other hand, I can understand @bnoordhuis opinion, I have barely ever seen other names for these parameters.